### DISCUSSION TITLE
Cipi v4, any update on the release?

### DISCUSSION THREAD

**Discussion by Sidgut** [andreapollastri/cipi#439] [General]
The best time to release Cipi v4 is now! Runcloud has also removed its Free plan from all customers (new & old), I am eager to hear the good news about Cipi v4.

Let me know if any help in required in preparing the Readme docs or something, open to contribute.

Thanks again for doing an awesome work! :)

**Comment by donniewr** [comment]
andreapollastri has made many contributions in private repo since July. I hope V4 will be released soon.

**Comment by fastmover** [comment]
If only they'd make it public, I would love to help.

**Comment by AhmedBinNasser** [comment]
Any coming updates?

**Comment by yolanmees** [comment]
Cipi hasn't had an update for over a year. So I'm trying to bring life to Cipi again. I forked Cipi to a new repository. I am working on a migration and easier update system first.

https://github.com/yolanmees/Spikster

I already added PHP 8.2 support and Ubuntu 22.04 / 23.04

**Reply by andreapollastri** [reply]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
